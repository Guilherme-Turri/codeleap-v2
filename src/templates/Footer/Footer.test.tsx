import { componentRender } from '../../render/componentRender'
import { Footer } from '.'
import { screen } from '@testing-library/react'

describe('<Footer/>', () => {
  it('should render on screen', () => {
    const { container } = componentRender(<Footer />)
    expect(screen.getByText('Guilherme Turri - 2023')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  })
})
