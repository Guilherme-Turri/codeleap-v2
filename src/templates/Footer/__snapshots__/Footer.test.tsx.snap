// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> should render on screen 1`] = `
.c0 {
  padding: 1.5rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-top: 3px solid;
  border-image: linear-gradient(90deg,rgba(44,62,80,0.1) 0px,rgba(44,100,80,0.6) 40%,rgba(44,62,80,0.1) 100%) 1;
  background-clip: padding-box;
  margin-top: 0.625rem;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div>
  <footer
    class="c0"
  >
    <p
      class="c1"
    >
      Guilherme Turri - 2023
    </p>
  </footer>
</div>
`;
